### ISSUE TITLE
GraphQL Cart endpoint returns incorrect or different product images

### ISSUE THREAD

**Issue by stijnbernards** [magento/magento2#26113]
### Preconditions (*)
1. Blank Magento 2.3.3 installation
2. No Sample Data installed
3. Uploaded custom images via backend.

### Steps to reproduce (*)
1. Create a guest cart via GraphQL
2. Add a product to your cart via GraphQL
3. Query the cart via GraphQL
```graphql
query {
  cart(
    cart_id: "3gw4rRy7CEEDqoTvm6ykqkGdERf2DdN4"
  ) {
    items {
      product {
        name
        image {
          label
          url
        }
      }
    }
  }
}

```

### Expected result (*)
1. Image url is working as it should

### Actual result (*)
1. Check the query cart response, the image is replaced by a placeholder and the url seems broken. This tells me there is something wrong with the CartItem Resolver. Has this been fixed in cart_items_v2?
![image](https://user-images.githubusercontent.com/6909174/69404771-57a6d800-0cfe-11ea-94f8-0ccabec6a9a1.png)
```graphql
{
  "data": {
    "cart": {
      "items": [
        {
          "product": {
            "name": "Sprod",
            "image": {
              "label": "Sprod",
              "url": "http://magento2.loc/pub/static/version1569835373/graphql/_view/en_US/Magento_Catalog/images/product/placeholder/image.jpg"
            }
          }
        }
      ]
    }
  }
}
```

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Bravo 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-29945` were created

**Issue Available**: @engcom-Bravo, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by karyna-t** [comment]
possible fix for a problem with a broken url for the placeholder
https://github.com/magento/magento2/pull/26386


To get the product image that was uploaded (not a placeholder), I guess you can add a corresponding attribute to the catalog_attributes.xml file.
e.g. in Custom/Module/etc/catalog_attributes.xml
```
<group name="quote_item">
    <attribute name="image"/>
</group>
```

**Comment by danielrenaud** [comment]
Thanks for your contribution. Internal Magento team is working on this issue right now.

**Comment by magento-engcom-team** [comment]
Hi @stijnbernards, @danielrenaud, @karyna-tsymbal-atwix. 

Thank you for your report and collaboration!

The issue was fixed by Magento team. The fix was delivered into `magento/magento2:2.4-develop` branch(es).
Related commit(s):
 - [1b8dfb27053cf03e98c19a0b47a98dfeb5e54459](https://github.com/magento/magento2/commit/1b8dfb27053cf03e98c19a0b47a98dfeb5e54459)
 - [6666ea5aeef42c60564f9fc11a9e9b920fb8f0c1](https://github.com/magento/magento2/commit/6666ea5aeef42c60564f9fc11a9e9b920fb8f0c1)
 - [218f3ab567bf9cc8c19a88a7d5d45d36558180f3](https://github.com/magento/magento2/commit/218f3ab567bf9cc8c19a88a7d5d45d36558180f3)
 - [50bf58ea9799ba9caf5d00d7c09dfe38f99c6210](https://github.com/magento/magento2/commit/50bf58ea9799ba9caf5d00d7c09dfe38f99c6210)
 - [d8b060b2e55a02b27a10b84ab26ff8a720c76fe2](https://github.com/magento/magento2/commit/d8b060b2e55a02b27a10b84ab26ff8a720c76fe2)

The fix will be available with the upcoming `2.4.0` release.
